migrations/20170304184220_add_constraints.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable quotes */
+
+exports.up = knex =>
+  knex.schema
+    .raw(`CREATE TYPE job_status AS ENUM ('pending', 'progress', 'complete', 'error')`)
```

Comment:
I'm wondering if we need PG constraints once we have the objections ones, but why not :).